.pre-commit-config.yaml

```
@@ -58,6 +58,7 @@ repos:
       - passlib
       - pytest~=3.6,<5.0.0
       - sphinx<4
+      - importlib_metadata~=4.6.1
```

```suggestion
      - importlib_metadata~=4.3
```

to be inline with https://github.com/aiidateam/aiida-core/pull/5058

and in setup.json

---

@chrisjsewell  I have made the changes